app/Repositories/Eloquent/AllocationRepository.php

```diff
@@ -42,17 +43,29 @@ public function getUnassignedAllocationIds(int $node): array
      */
     protected function getDiscardableDedicatedAllocations(array $nodes = []): array
     {
-        $query = Allocation::query()->selectRaw('CONCAT_WS("-", node_id, ip) as result');
+        if (DB::getDriverName() === 'pgsql') {
```

Comment:
Either I'm blind, or the query is exactly the same.

Reply:
The quotations on the `CONCAT_WS` separator are different.

Reply:
My bad.